hypha/apply/projects/models/project.py

```diff
@@ -207,14 +209,17 @@ def create_from_submission(cls, submission):
 
         # See if there is a form field named "legal name", if not use user name.
         legal_name = submission.get_answer_from_label('legal name') or submission.user.full_name
-
+        vendor, _ = Vendor.objects.get_or_create(
+            user=submission.user
+        )
+        vendor.name = legal_name
+        vendor.address = submission.form_data.get('address', '')
+        vendor.save()
```

Comment:
>Does this not make it impossible to create a vendor for a new project since VendorAccessMixin is used for CreateVendorView?

@frjo, we initialise vendor on project creation, to save vendor related data directly there instead of project.

So a project existing without vendor would not be valid.

Reply:
Missed that, nice!

Reply:
@frjo, one more point, as we moved the contact fields from project to vendor, we don't yet have address and phone in vendor edit form which was there earlier in [Project edit form](https://github.com/HyphaApp/hypha/blob/c72be985e075e2594e279e6f3c4e65317814662b/hypha/apply/projects/forms.py#L150).

Do we include these two fields in the vendor form as well?

Reply:
Wait with that until someone say they really need it.